recipes/qwt/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+import os
+from conans import ConanFile, tools
+
+
+class QwtConan(ConanFile):
+    name = "qwt"
+    license = "Qwt License, Version 1.0"
```

Comment:
Reading it https://qwt.sourceforge.io/qwtlicense.html

```suggestion
    license = "LGPL-2.1-or-later"
```